packages/yew/src/server_renderer.rs

```diff
@@ -9,6 +9,31 @@ use crate::html::{BaseComponent, Scope};
 use crate::platform::fmt::BufStream;
 use crate::platform::{LocalHandle, Runtime};
 
+/// Passed top-down as context for `render_into_stream` functions to know the current innermost
+/// VTag kind to apply appropriate text escaping.
+#[cfg(feature = "ssr")]
```

Comment:
Can you put these items in a module and feature gate the entire mod? That's how it's done elsewhere in the code base. See the `feat_*` modules